pyproject.toml

```diff
@@ -0,0 +1,72 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs >= 0.3.0"]
+build-backend = "hatchling.build"
+
+[tool.hatch.version]
+path = "marda_registry/__init__.py"
+
+[tool.hatch.metadata]
+# required to allow git deps in optional dependencies
+allow-direct-references = true
+
+[tool.hatch.build.targets.wheel]
+packages = ["marda_registry"]
+
+[project]
+name = "marda-extractors-registry"
+readme = "README.md"
+dynamic = ["version"]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Development Status :: 2 - Pre-Alpha",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: System Administrators",
+    "Intended Audience :: Information Technology",
+    "Operating System :: OS Independent",
+    "Topic :: Other/Nonlisted Topic",
+    "Topic :: Scientific/Engineering",
+]
+requires-python = ">=3.10"
+dependencies = [
+    "fastapi",
+    "httpx",
+    "invoke",
+    "jinja2",
+    "linkml ~= 1.6, != 1.6.2",
+    "linkml-runtime ~= 1.6, != 1.6.1",
```

Comment:
We should check that we're fine with `linkml-runtime == 1.6.3` and `linkml == 1.6.9` which is the latest. But I guess that's for `schema`.